docs/pages/serving_an_origin.mdx

```diff
@@ -55,123 +57,309 @@ Once you go through the process, locate your credential files and set the follow
+* `<federation URL>` is the federation URL discussed above
+* `</path/to/data>` is the absolute path to the directory containing files you want to export as Pelican objects in your federation
+* `</your/federation/prefix>` is the federation prefix at which files in `/path/to/data` will be accessed from in the federation. Note that federation prefixes follow POSIX path conventions, and they must begin with `/` to denote an absolute path.
+
+> **NOTE:** By default, origins require authorization tokens for object access. There's currently no way to serve a public origin using only the command line, but you can find more information about various access controls by looking at [origin capabilities](#origin-and-namespace-capabilities) below.
+
+To run the same origin using a `pelican.yaml` configuration file, save your configuration to `/etc/pelican/pelican.yaml` if you're running Pelican as root, and at `~/.config/pelican/pelican.yaml` if you're running as a non-root user. The command line origin from above could be configured accordingly:
+
+```
+# Tell Pelican which federation you're joining
+Federation:
+  DiscoveryUrl: <federation URL>
+
+# Configure your Origin
+Origin:
+  # POSIX is the default storage type for Pelican origins
+  # and can be omitted
+  StorageType: "posix"
+
+  Exports:
+    - StoragePrefix: "/path/to/data"
+      FederationPrefix: "/your/federation/prefix"
+      # Explicitly state what capabilities you want this prefix to have
+      Capabilities: ["Reads", "Writes"]
+
+```
+and then simply run
+```bash
+pelican origin serve
+```
+Pelican will read the config file and apply it to your origin.
+
+
+Finally, origins can be configured to a limited extent with environment variables. In Pelican's environment variable model, configuration options are taken from `pelican.yaml`, flattened, and prepended with either `PELICAN_` or `OSDF_`, depending on the name of the binary you're using (ie whether you run `osdf serve` or `pelican serve` commands).
 
-This will start Pelican origin as a daemon process.
+For example, you might configure the origin's storage type by setting the environment variable `PELICAN_ORIGIN_STORAGETYPE=posix`.
+
+> Environment variable configuration does not support the same complex structures that can be built with yaml configuration, such as lists.
 
 The first time the origin is started, you will see something that looks like the following:
 
 ```console
-$ pelican origin serve -f osg-htc.org -v $PWD:/demo
+$ pelican origin serve -f https://osg-htc.org -v $PWD:/demo
 
 Pelican admin interface is not initialized
 To initialize, login at https://localhost:8444/view/initialization/code/ with the following code:
 551220
 ```
+See the [browser configuration](#login-to-admin-website) documentation section for more information about initializing your origin's web interface.
+
+### Origin and Namespace Capabilities
```

Comment:
Would it worth moving this section and Multi-Export origin to a separate page like "Configuring Origin"? to make the documentation easier to follow?